main.go

```diff
@@ -92,7 +88,7 @@ func run(ctx context.Context, wg *sync.WaitGroup, globalCollectionOpts state.Col
 		if config.SystemType == "google_cloudsql" {
 			hasAnyGoogleCloudSQL = true
 		}
-		if config.SystemType == "heroku" {
+		if config.SystemType == "heroku" && os.Getenv("DYNO") != "" && os.Getenv("PORT") != "" {
```

Comment:
? (not objecting, just curious about the change)

Reply:
We were previously making the same check further down - essentially it tries to protect against situation where we incorrectly identify something as being a Heroku system type.

That said, we don't actually need to verify that, because we have the same check already in https://github.com/pganalyze/collector/blob/main/config/read.go#L473 -- so I'll drop this.